test/utils/v1beta1/controllers.go

```diff
@@ -18,12 +19,25 @@ import (
 // but it disables the prometheus metrics and leader election. This intended
 // to for testing.
 func StartBuildControllers() (*bytes.Buffer, error) {
+	// Check for BUILDRUN_EXECUTOR environment variable
+	executor := os.Getenv("BUILDRUN_EXECUTOR")
+	return StartBuildControllersWithExecutor(executor)
+}
+
+// StartBuildControllersWithExecutor initialize an operator with a specific executor type.
+// If executor is empty, it will use the default or environment variable value.
+func StartBuildControllersWithExecutor(executor string) (*bytes.Buffer, error) {
 	logBuffer := &bytes.Buffer{}
 	l := ctxlog.NewLoggerTo(logBuffer, "controller")
 	ctx := ctxlog.NewParentContext(l)
 
 	c := buildconfig.NewDefaultConfig()
 
+	// Set executor if provided
```

Comment:
Calling `SetConfigFromEnv` later changes again the BuildrunExecutor, does not it?